website/blog/2021-03-05-GAAD-React-Native-Accessibility.md

```diff
@@ -0,0 +1,33 @@
+---
+title: The GAAD Pledge - Improving React Native Accessibility
+author: Alexandra Marlette
+authorTitle: GAAD Pledge Open Source Accessibility Community Manager for React Native
+authorURL: 'https://twitter.com/alexmarlette'
+authorImageURL: 'https://avatars.githubusercontent.com/u/10052470?s=460&u=7f2304cb929d1de703856717af86324c66728f3a&v=4'
+authorTwitter: alexmarlette
+tags: [announcement]
+---
+
+## Hello React Native Community,
+
+In May 2020 Facebook was the first company to take the [GAAD pledge](https://diamond.la/GAADPledge/), by doing so they committed to making accessibility a core part of the React Native open source project. Since May, Facebook has spent that time thoughtfully reviewing and documenting accessibility gaps within React Native. So far the gap analysis has surfaced 90 issues, all of which have been translated to Github issues on [React Native's Github](https://github.com/facebook/react-native/projects/15).
+
+Overall, we found that React Native APIs provide strong support for accessibility. However, we also found many core components do not yet fully utilize platform accessibility APIs and support is missing for some platform specific features.
+
+With knowledge of where the gaps exist, work can begin on closing them, the enthusiasm and diversity of contributors have always played a critical role in the development of React Native and whether a seasoned or new contributor, we're asking you to join in making React Native more accessible.
```

Comment:
What about this? The original form felt a bit formal and difficult to read for me. 

```suggestion
We have found that many of these gaps in accessibility are great opportunities for new contributors to React Native. If you have been interested in contributing to React Native, we encourage you to join us in making React Native more accessible. 
```

Reply:
My main concern is that we were also trying to convey that the community has played a large part in previous developments and that gets missed in the suggestion. Maybe:

The enthusiasm and diversity of contributors have always played a critical role in the development of React Native and these gaps in accessibility are great opportunities for current and new contributors. If you have been interested in contributing to React Native, we encourage you to join us in making React Native more accessible.